src/js/state/Launches/index.ts

```diff
@@ -10,9 +10,8 @@ const slice = createSlice({
   name: "$Launches",
   initialState: {},
   reducers: {
-    touchVersion: (state, action: PayloadAction<string | undefined>) => {
-      let key = action.payload || getVersion()
```

Comment:
This getVersion() call was what was causing the error. It makes a sync ipc call to the 'main' process. But this code was being run in the "main" process.